docker/airflow-python/Dockerfile

```diff
@@ -4,12 +4,14 @@ FROM python:${PYTHON_VERSION}-slim as airflow-base
 ENV AIRFLOW_BUILD_DEPS="freetds-dev python-dev libkrb5-dev libssl-dev libffi-dev libpq-dev git"
 ENV AIRFLOW_APT_DEPS="libsasl2-dev sasl2-bin libsasl2-2 libsasl2-dev libsasl2-modules freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
 
-ENV AIRFLOW_VERSION=1.10.3
+ENV AIRFLOW_VERSION=1.10.4
```

Comment:
I would advise to upgrade to 1.10.5 right away. Airflow 1.10.4 has some regression issues.

Reply:
Done